bin/metadata-json-lint

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+#
+# This file was generated by Bundler.
+#
+# The application 'metadata-json-lint' is installed as part of a gem, and
+# this file is here to facilitate running it.
+#
+
+require "pathname"
+ENV["BUNDLE_GEMFILE"] ||= File.expand_path("../../Gemfile",
+  Pathname.new(__FILE__).realpath)
+
+require "rubygems"
+require "bundler/setup"
+
+load Gem.bin_path("metadata-json-lint", "metadata-json-lint")
```

Comment:
this looks like the bin/ directory that's created by the PDK. these files don't need to be committed.

Reply:
Not sure where this came from. Will delete.